src/FileUtils/AgGridUtils.js

```diff
@@ -95,9 +83,9 @@ const _validateFormat = (rows, hasHeader, cols, options) => {
   const startIndex = hasHeader ? 1 : 0;
   for (let rowIndex = startIndex; rowIndex < rows.length; rowIndex++) {
     const row = rows[rowIndex];
-    while (row[row.length - 1] === undefined && row.length > colsData.length) row.pop();
+    while (row[row.length - 1] === undefined && row.length > knownColsCount) row.pop();
     if (row.length !== knownColsCount || row.includes(undefined))
-      _forgeColumnsCountError(row, rowIndex, colsData, errors);
+      _forgeColumnsCountError(row, rowIndex + 1, colsData, errors);
```

Comment:
If you compute `rowIndex + 1` here to get the line number to display in the error message, you should rename the argument in `_forgeColumnsCountError` and `_forgeTypeError` from `rowIndex` to something like `rowLineNumber`, to avoid confusion with the row **index** in the array of all rows.

Reply:
Done :+1: